deploy/backplane/mtsre/ocs-consumer/01-mtsre-ocs-consumer-cr-admins.ClusterRole.yml

```diff
@@ -0,0 +1,51 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: backplane-mtsre-ocs-consumer-cr-admins
+rules:
+  - apiGroups:
+    - "ocs.openshift.io"
```

Comment:
question for all of these: are these namespaced or cluster-scoped CRDs?

Reply:
I can confirm although would that create any difference in our case?
I ask this because as to my knowledge, it won't be feasible to define `Role` because we're going to be feed them to the `SubjectPermission` and SubjectPermission expects `ClusterRole` to be specified under it, not Role.

And to scope ClusterRole to the bounds of namespaces, we need to specify `RoleBinding` but we don't specify them, rather SubjectPermission does that for us behind the scenes via the `namespacesAllowedRegex`.

Please correct me if I am wrong anywhere :)

Thanks!

Reply:
Yes, it makes a difference to how the SubjectPermission applies.